source/fundamentals/data-formats/document-data-format-extended-json.txt

```diff
@@ -33,7 +33,7 @@ Extended JSON Formats
 ---------------------
 
 MongoDB Extended JSON features different string formats to represent BSON data.
-Each of the different formats conform to the `JSON RFC <https://tools.ietf.org/html/rfc7159>`__
```

Comment:
Issue: I do not believe this link should not be extracted

Reply:
I disagree upon review. This is a distractor link that would result in the user navigating away to a page that does not present an alternative to the information on this page.